test/packages/good/data_stream/foo/_dev/deploy/tf/env.yml

```diff
@@ -2,8 +2,5 @@ version: '2.3'
 services:
   terraform:
     environment:
-      - AWS_ACCESS_KEY_ID=${AWS_ACCESS_KEY_ID}
-      - AWS_SECRET_ACCESS_KEY=${AWS_SECRET_ACCESS_KEY}
-      - AWS_SESSION_TOKEN=${AWS_SESSION_TOKEN}
-      - AWS_PROFILE=${AWS_PROFILE}
-      - AWS_REGION=${AWS_REGION:-us-east-1}
\ No newline at end of file
+      - FOO=${BAR:-default-value}
```

Comment:
Unrelated changes?

Reply:
No but given the file is not required I'll remove it